azure-pipelines.yml

```diff
@@ -0,0 +1,106 @@
+name: ${{ parameters.Action }} - ${{ parameters.env }}.${{ parameters.location }} 
+
+trigger: none
+
+parameters:
+
+  - name: Action
+    displayName: Action
+    type: string
+    default: 'Plan'
+    values:
+    - Plan
+    - Apply
+    - Destroy
+
+  - name: location
+    displayName: Location
+    type: string
+    default: 'UK South'
+    values:
+    - 'UK South'
+    - 'UK West'
+
+  - name: env
+    displayName: Environment
+    type: string
+    default: 'PTLSBOX'
+    values:
+    - PTLSBOX
+
+variables:
+  - name: tfversion
+    value: 0.15.4
+  - name: project
+    value: ss
+  - name: serviceConnection
+    value: OPS-APPROVAL-GATE-${{ parameters.env }}-ENVS
+  - name: tfInitSub
+    value: "04d27a32-7a07-48b3-95b8-3c8691e1a263"
+  - name: product
+    value: 'sds-jenkins'
+  - name: action
+    value: ${{ parameters.Action }}
+  - name: builtFrom
+    value: $(Build.Repository.Name)
+
+stages:
+  - stage: 
+    displayName: 'Jenkins Infrastructure'
+    jobs:
```

Comment:
this should use the standard pipeline from https://github.com/hmcts/cnp-azuredevops-libraries

Reply:
Updated to match standard